TypeSystem/Schema/smoke_test_struct.h

```diff
@@ -20,6 +21,7 @@ CURRENT_STRUCT(Primitives) {
   CURRENT_FIELD(m, bool);
   CURRENT_FIELD(n, std::chrono::microseconds);
   CURRENT_FIELD(o, std::chrono::milliseconds);
+  CURRENT_FIELD_DESCRIPTION(b, "Field descriptions can be set in any order.");
```

Comment:
Done, 493832f.
A bit messy in plain C struct output format, but should do it for now.